fhservices.cfg.j2

```diff
@@ -74,3 +79,15 @@ define service {
        contact_groups rhmapadmins
 }
 {% endfor %}
+
+{% for chk in ['usage','inodes'] %}
+define service {
+       service_description Storage::Pod::Disk {{ chk | capitalize() }}
+       hosts {{ rhmap_router_dns }}
+       check_command check_pod_disk_usage!{{ chk }}!80!90
```

Comment:
Yeah.  I'm with you.  Our current checks in our SaaS platform are separate ones, but I see no reason why we can't get the data for both pieces in one check.  We should probably keep separate thresholds for parameters just in case we decide to do it differently.